src/polyfills.js

```diff
@@ -42,6 +44,10 @@ if (self.document) {
   // The anonymous class parameter allows us to detect native classes vs
   // transpiled classes.
   installCustomElements(self, class {});
+  // eslint-disable-next-line no-undef
+  if (INTERSECTION_OBSERVER_POLYFILL || getMode().localDev || getMode().test) {
```

Comment:
Are we going to remove `getMode().test` before merging? If not does it mean all our tests are testing against the new changes.

Reply:
That's the idea - yes.

Reply:
But if I find that too many tests are failing, I might reconsider this part.

Reply:
I see. Cool